gpkit/constraints/set.py

```diff
@@ -5,6 +5,16 @@
 from ..repr_conventions import _str, _repr, _repr_latex_
 
 
+def _sort_by_num_models(var):
+    "return integer for Variable sorting"
+    return len(v.key.models) if v.key.models else 0
```

Comment:
possible premature optimization, but would it be faster to not lookup `v.key.models` twice?

```python
def _sort_by_num_models(var):
    "return integer for Variable sorting"
    mods = v.key.models
    return len(mods) if mods else 0
```